assets/js/base/components/button/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { Button as WPButton } from '@wordpress/components';
+import { Button as WPButton } from 'wordpress-components';
```

Comment:
> This means the direct import would be used in the editor as well (via our builds) right?

Yes, the consequence of loading these components from the aliased version is that we are loading `@wordpress/components` twice in the editor: the external and our aliased version.

That also means we are loading the styles twice:

![imatge](https://user-images.githubusercontent.com/3616980/71994902-e2ee1580-3239-11ea-8811-d7ec222733ff.png)

That's far from the ideal solution, but I'm wary about only loading one stylesheet, because between different version one component might have been added or removed, so we can't reliably trust one of the stylesheets only.

A CSS-in-JS approach like the one used in the `<Card />` component, would solve that. But several components are not using it yet.

So technically there could be conflicts in the editor (not in the frontend, where we are only loading the aliased version), but I don't know how common that would be.

> we could probably introduce an alias that resolves to the direct import on frontend builds and an external on backend builds (via webpack config, since we separate out those builds), but I'm on the fence about whether it's overkill. If the first point above turns out not being an issue then this won't be.

That wouldn't work either because even in the editor we need to import the aliased version of `@wordpress/components` in some cases: for example when showing the _Cart_ block, which includes a `<Card />` component, we will always need the aliased version of `@wordpress/components`, no matter if it's in the frontend or the editor.

I can't think of any way to reliably load only one version of `@wordpress/components` in the editor, since the Gutenberg layout will be written based on one version that might be different from the version of our aliased package.